packages/coin-support-utils/tests/02.createAccount/05.makeCreateAccountsObservable/index.ts

```diff
@@ -0,0 +1,210 @@
+import { ICreateAccountEvent } from '@cypherock/coin-support-interfaces';
+import { describe, test } from '@jest/globals';
+import { Observer, Subscription } from 'rxjs';
+import { makeCreateAccountsObservable } from '../../../src';
+import {
+  createAccountParams,
+  createAppMock,
+  db,
+  getAddressesFromDeviceImplementation,
+  getAddressesFromDeviceMock,
+  testApp,
+} from '../../__mocks__';
+
+/**
+ * @todo Increase branch coverage
+ */
+describe('makeCreateAccountsObservable', () => {
+  beforeAll(() => {
+    jest.useFakeTimers();
+  });
+
+  beforeEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+    jest.resetAllMocks();
+  });
+
+  afterAll(() => {
+    jest.useRealTimers();
+  });
+
+  test('should create new accounts', done => {
+    db.account.getAll.mockResolvedValue([]);
```

Comment:
We need this test to be configurable. Currently only one test case of this kind can be ran which is hardcoded in `__mocks__/createAccountParams`